pyproject.toml

```diff
@@ -11,6 +11,8 @@ python = "^3.9"
 jsonschema = "^3.2.0"
 PyYAML = "^6.0"
 requests = "^2.28.1"
+gql = "^3.4.0"
+requests-toolbelt = "^0.10.0"
```

Comment:
Curious, why not requests-mock directly? https://pypi.org/project/requests-mock/